src/Lessons-api.Data/Repositories/TeacherRepository.cs

```diff
@@ -19,14 +20,14 @@ public TeacherRepository(LessonsContext lessonsContext)
 
         public async Task<TeacherEntity> GetTeacherById(int id)
         {
-            var result = await _lessonsContext.Teachers.FindAsync(id);
+            var result = await _lessonsContext.Teachers.Include(s => s.User).SingleOrDefaultAsync(s => s.Id == id);
```

Comment:
Change this to FirstOrDefaultAsync

![image](https://user-images.githubusercontent.com/64907389/128723910-174c7913-77eb-4852-b8e5-78aa23e7ee7e.png)